src/main/scala/com/plameno/validation/CommonValidators.scala

```diff
@@ -23,12 +25,16 @@ case class MaxLengthValidator(s: String, maxLen: Int, customError: Option[String
   def isValid = s.length <= maxLen
 }
 
-// TODO: RegexValidator with compiled pattern. And reuse it for email validation.
+case class RegexValidator(s: String, re: Regex, customError: Option[String] = None)
+  extends Validator(customError) {
+  override val defaultError = s"Regular expression ${re} could not be matched"
```

Comment:
Quote ${re} with quotes '' to have more user friendly message:
`s"Regular expression '${re}' could not`...